java/device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/Message.java

```diff
@@ -62,11 +61,6 @@
     private Boolean httpBatchSerializeAsString;
 
     /// <summary>
-    /// [Optional] Used when batching on HTTP Default: UTF-8.
-    /// </summary>
-    private StandardCharsets httpBatchEncoding;
-
```

Comment:
because StandardCharsets has not defined in Java 1.6 and this object has not been used in this class